### ISSUE TITLE
Missing sell ragfair

### ISSUE THREAD

**Issue by rikato** [matias-kovero/tarkov#3]
Im writing it myself now but would be nice if it's included

**Comment by rikato** [comment]
I was wondering what the api expects when offering. Does is expect an requirement per item you put up in one offer? Or one requirement for all items you put up?

**Comment by rikato** [comment]
`
{
  "data": [
    {
      "Action": "RagFairAddOffer",
      "sellInOnePiece": false,
      "items": [
        "itemidhere",
        "itemidhere",
        "itemidhere",
      ],
      "requirements": [
        {
          "requirement": {
            "_tpl": "itemidhere",
            "count": 10000
          },
          "level": 0,
          "side": 0,
          "onlyunctional": false
        }
      ]
    }
  ],
  "tm": 2
}
`
this request gives me an 228 error saying: RagFairAddOffer: not found templateId in "requirements" param" but thats the _tpl right? Am I missing something?

**Comment by matias-kovero** [comment]
Just released version `0.1.0` and it has this feature.

___
> `{ "data": [ { "Action": "RagFairAddOffer", "sellInOnePiece": false, "items": [ "itemidhere", "itemidhere", "itemidhere", ], "requirements": [ { "requirement": { "_tpl": "itemidhere", "count": 10000 }, "level": 0, "side": 0, "onlyunctional": false } ] } ], "tm": 2 }`
> this request gives me an 228 error saying: RagFairAddOffer: not found templateId in "requirements" param" but thats the _tpl right? Am I missing something?

The request is invalid. Should be:
```javascript
data: [{
  Action: "RagFairAddOffer",
  sellInOnePiece: false || true,
  items: [array_of_item_ids],
  requirements:[{
    _tpl: _tpl_Here,
    count: amoutHere,
    level: 0,
    side: 0,
    onlyFunctional: false,
  }],
  tm: 2,
}]
```
